src/App.js

```diff
@@ -1,12 +1,183 @@
 import React from 'react';
+
+import Game from './Components/Game/Game';
+// import Countdown from './Components/CountdownTimer/Countdown2';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Fast Fingers </h1>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.initialState = {
+      gameOn: false,
+      playerName: '',
+      selectedLevel: 'default',
+      playerNameError: '',
+      selectedLevelError: '',
+    };
+    this.state = { ...this.initialState };
+  }
+
+  // handles the change in difficulty level select element
+
+  setLevel = (event) => {
+    const value = event.target.value;
+    this.setState({
+      ...this.state,
```

Comment:
When you do setState() call, React merges the new state with the previous one
you don't need to pass `...this.state` explicitly in setState() call, React will take care of this.